### ISSUE TITLE
notification event not fired for remote notifications when app fresh open (not on background)

### ISSUE THREAD

**Issue by diegomichell** [CatalystCode/react-native-azurenotificationhub#156]
## Report

- [ ] I have searched [existing issues](https://github.com/CatalystCode/react-native-azurenotificationhub/issues) and this is not a duplicate.

## Environment

- Platforms: iOS

- Android version: 

- iOS version: 13.5

## Issues and Steps to Reproduce
1. Register for notifications and fully close the app
2.You will still get notifications but when you click on them app will open  'notification' event won't fire.

## Expected Behavior
-'notification' event should be fire.

## Actual Behavior
-You received a notification with the app fully closed (not in background) and it will take you to your app main screen and won't execute any custom logic you had put on your listener for 'notification' event.

## Link to Code
```
  // notification: Fired when a remote notification is received. This won't work
  IOSNotificationHub.addEventListener('notification', this._handlerRemoteNotification);

 // notification: Fired when app is fresh opened from remote notification. This is a current workaround you can use, call getInitialNotification on your app initial load and check whether there is a notification.
    IOSNotificationHub.getInitialNotification().then(notification => notification && this._handlerRemoteNotification(notification));
```

**Comment by hosseinjalili** [comment]
did you installed this package @react-native-community/push-notification-ios  or this react-native-push-notification ? 
because this issue #159

**Comment by diegomichell** [comment]
Yes I had installed @react-native-community/push-notification-ios in the past, but by the time I got this error I didn't have it on my dependencies anymore.

**Comment by kumarparth380** [comment]
I am also having the same issue as `notification event not fired for remote notifications when app fresh open (not on background)` also I am unable to fetch the payload from notification but couldn't do so. I tried putting debugger on the AppDelegate.m code for notification even and surprising it doesn't go on that point if notification is received on background and opened.

**Comment by ramonvermeulen** [comment]
I am currently facing exactly the same issue on iOS only. The eventlistener on notification is not fired from a clean boot. Only when the app is running in the background the event is fired for me on iOS. How can I get the initial notification?

**Comment by olie-chan** [comment]
Any update on this?

We have an existing app that used to get the notification event fired on ios when in the foreground which doesn't seem to be triggered anymore. I wonder if it's possible that IOS changed the notification behaviour?
